app/javascript/src/components/Dashboard/Contacts/Table/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Table as NeetoUITable, Typography, Avatar } from "neetoui";
+
+import { CONTACT_TABLE_COLUMN_DATA } from "../constants";
+import { stringToDate } from "../utils";
+
+const Table = ({
+  selectedContactIds,
+  setSelectedContactIds,
+  contacts = [],
+}) => {
+  const modifyContactDateFormat = contacts =>
+    contacts.map(contact => ({
+      ...contact,
+      created_at: stringToDate(contact["created_at"]),
+    }));
+  const renderMenuColumn = () => <MenuHorizontal size={16} />;
+  const renderProfileColumn = (name, { role, profile_pic }) => (
```

Comment:
We should be using camelCase for all variables names and further manipulations of backend data once it reaches front-end side.

In our neeto ecosystem this is automatically handled by our Axios interceptors where case conversion is done. But here in your codebase you will have to do it manually for the time being.

Reply:
Okay, Yedhin I will manually convert the data keys to camel case. Thanks for pointing this out.

Reply:
I have removed all occurrences of snake case and replaced it with camel case. Kindly ignore it if it appears in any of the PRs that are already made.